ODPC.Server/Features/Publicaties/PublicatiesMock.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace ODPC.Features.Publicaties
+{
+    public class Publicatie
+    {
+        public Guid Uuid { get; set; }
+        public string? OfficieleTitel { get; set; }
+        public string? VerkorteTitel { get; set; }
+        public string? Omschrijving { get; set; }
+        public DateOnly Creatiedatum { get; set; }
+        public string? Status { get; set; }
+    }
+
+    [ApiController]
+    [Route("api/v1/publicaties")]
+    public class PublicatiesMock: ControllerBase
```

Comment:
ik zou het niet eens een mock noemen. hier gaan we op verder werken. dit is gewoon de eerste versie.

Reply:
ik zou hem ook meteen in features splitsen

Reply:
Goede suggesties, heb ik doorgevoerd